pages/u/[username]/index.tsx

```diff
@@ -121,7 +122,7 @@ export const getServerSideProps = async (context: UserSSRPropsContext) => {
   }
 
   const userData = (await req.json()) as DbUser;
-  const ogImage = `${process.env.NEXT_PUBLIC_OPENGRAPH_URL}/users/${username}`;
+  const ogImage = siteUrl(`og-images/dev-card`, { username });
```

Comment:
This adds the Dev card OG image as the OG image for the user profile page now as requested @brandonroberts.

Reply:
<img src="https://media1.giphy.com/media/T9JtEyoJ43gY4wLOqW/giphy.gif"/>